Core/include/Acts/Visualization/GeometryView3D.hpp

```diff
@@ -103,16 +103,19 @@ struct GeometryView3D {
   /// @param [in,out] helper The visualization helper
   /// @param volume The DetectorVolume to be drawn
   /// @param gctx The geometry context for which it is drawn
+  /// @param drawSurfaces switches on/off the drawing of the volumes' surfaces
   /// @param transform An option additional transform
   /// @param connected The config for connected portals
   /// @param unconnected The config for unconnected portals
+  /// @param ViewConfig The drawing configuration
```

Comment:
```suggestion
  /// @param viewConfig The drawing configuration
```

apparently this was upper case where I copied it from :facepalm: one more time @dimitra97